wordpress-mobile/WordPressUI-iOS
# Revert workaround and rename WordPressUI.bundle to WordPressUIResources.bundle to avoid conflicts

@jkmassel implemented a temporary workaround for https://github.com/wordpress-mobile/WordPress-iOS/issues/11848 in https://github.com/wordpress-mobile/WordPressUI-iOS/pull/36 by generating all images in code, circumventing the need to load images from the resource bundle.

This PR reverts that workaround and fixes the underlying issue. I outlined the cause in this comment: https://github.com/wordpress-mobile/WordPress-iOS/issues/11848#issuecomment-499825738. The fix is to rename `WordPressUI.bundle` to `WordPressUIResources.bundle` to avoid iOS trying to load the images from `WordPressUI.framework`.

The changes I have made to address the issue are in this commit: https://github.com/wordpress-mobile/WordPressUI-iOS/pull/37/commits/b2e98891de30e5d7ac989c7410b192c68c08b655.

### To Test

1. See that the checks on this PR pass (resource loading still works outside CocoaPods).
2. On my WPiOS branch, `try/image-crash` with tests for loading the WordPressUI images, see that it [crashes without this fix](https://circleci.com/gh/wordpress-mobile/WordPress-iOS/8302) and [passes with the fix](https://circleci.com/gh/wordpress-mobile/WordPress-iOS/8303). Feel free to checkout the branch and run the tests yourself.
